libafl/src/monitors/statsd.rs

```diff
@@ -0,0 +1,208 @@
+//! StatsD monitor.
+//!
+//! This roughly corresponds to the [AFL++'s rpc_statsd](https://github.com/AFLplusplus/AFLplusplus/blob/stable/docs/rpc_statsd.md),
+//! so you could view such documentation for detailed information.
```

Comment:
We could copy&paste some of the docu here/into the libafl book - the referred doc could change, too